macros/models/_sector/transfers/transfers_native.sql

```diff
@@ -0,0 +1,182 @@
+{% macro transfers_native(blockchain, traces, transactions, native_token_address, genesis_balances=null, native_erc_transfers=null) %}
+
+WITH
+
+native_transfers  as (
+        SELECT 
+            'receive' as transfer_type, 
+            tx_hash,
+            trace_address, 
+            block_time,
+            to as wallet_address, 
+            {{native_token_address}} as token_address,
+            CAST(value as double) as amount_raw
+        FROM 
+        {{ traces }}
+        WHERE (call_type NOT IN ('delegatecall', 'callcode', 'staticcall') OR call_type IS NULL)
+        AND success
+        AND CAST(value as double) > 0
+        AND to IS NOT NULL 
+        AND to != 0x0000000000000000000000000000000000000000
+        {% if is_incremental() %}
+            AND block_time >= date_trunc('day', now() - interval '3' Day)
+        {% endif %}
+
+        UNION ALL 
+
+        SELECT 
+            'send' as transfer_type, 
+            tx_hash,
+            trace_address, 
+            block_time,
+            "from" as wallet_address, 
+            {{native_token_address}} as token_address,
```

Comment:
is this inteded to be `0xeeeeeeee....`, `0xDeadDeAddeAddEAddeadDEaDDEAdDeaDDeAD0000`, or something else?

Reply:
yeah, depends on what the wrapped native token address is, it should be the same with the native token transferred in erc20 events (for optimism and celo e.g)